src/slicing.erl

```diff
@@ -0,0 +1,71 @@
+%% This module finds the functions that will be tested with random data
+%% based on the diff output of the two commits and the type of refactoring
+%% that was done
+-module(slicing).
+
+-export([scope/2]).
+
+-type fun_info() :: {atom(), string(), integer()}.
+
+% These all have to be called from the directory of the source code! TODO, pass the folder as argument?
+
+-spec scope(string(), [string()]) -> {[{atom(), atom(), proper_types:type()}]}.
+scope(Diff, Sources) ->
+    Hunks = diff:diff(Diff, Sources),
+
+    NewFuncs = new(Hunks),
+    RemovedFuncs = removed(Hunks),
```

Comment:
After thinking about these a bit, I'm not sure if we need to consider functions that were either removed from the source, or added to it.
I assume that these changes mean that in other function(s) they were either inlined, or extracted.
Which means that I'm already considering these callers as modified functions in `ModifiedFuncs`.
Is this assumption correct, or there are other cases I haven't tought of?